requirements/requirements-no-engine.yml

```diff
@@ -35,3 +35,4 @@ dependencies:
       - tqdm
       - git+https://github.com/airspeed-velocity/asv.git@ef016e233cb9a0b19d517135104f49e0a3c380e9
       - connectorx>=0.2.6a4
+      - redis
```

Comment:
What's need of dependence from `redis` in this file?

Reply:
Because we use Ray alongside with this file for testing Ray master.